speedscale/README.md

```diff
@@ -0,0 +1,56 @@
+# Agent Check: Speedscale
+
+## Overview
+
+This integration publishes traffic replay results from [Speedscale][1] into Datadog. This lets you combine your observability data from Datadog with the results of a particular replay to investigate the root cause of poor performance. Find and troubleshoot potential performance issues before they show up in production with a combination of Speedscale and Datadog.
+
+## Setup
+
+### Configuration
+
+1. To use this integration you need a Datadog [API Key][2] so that the Datadog Agent can submit metrics and events to Datadog.
```

Comment:
```suggestion
1. To use this integration you need a Datadog [API Key][2] so that metrics and events can be submitted into Datadog.
```
Correct me if I'm wrong, but I thought that this integration did not use the Datadog agent?